app/bundles/CoreBundle/Test/MauticMysqlTestCase.php

```diff
@@ -127,12 +127,24 @@ private function applySqlFromFile($file)
     {
         $connection = $this->connection;
         $password   = ($connection->getPassword()) ? " -p{$connection->getPassword()}" : '';
-        $command    = "mysql -h{$connection->getHost()} -P{$connection->getPort()} -u{$connection->getUsername()}$password {$connection->getDatabase()} < {$file} 2>&1 | grep -v \"Using a password\" || true";
+        $command    = "mysql -h{$connection->getHost()} -P{$connection->getPort()} -u{$connection->getUsername()}$password {$connection->getDatabase()} < {$file}";
 
-        $lastLine = system($command, $status);
+        // 0 -> stdin, 1 -> stdout, 2 -> stderr
+        // We only need stderr, in case we need to output errors when the result code != 0.
+        $descriptors = [
+            2 => ['pipe', 'w'],
+        ];
+        $pipes       = [];
+
+        $process = proc_open($command, $descriptors, $pipes);
+        $stderr  = stream_get_contents($pipes[2]);
+        fclose($pipes[2]);
+        $result_code = proc_close($process);
 
-        if (0 !== $status) {
-            throw new Exception($command.' failed with status code '.$status.' and last line of "'.$lastLine.'"');
+        $lastLine = system($command, $result_code);
```

Comment:
Please move the `$lastLine` variable inside the next `if` statement, because it's used only there.
If you add MYSQL_PWD then the `$command` need to have an env variable 
```php
$lastLine = system('export MYSQL_PWD="'.$connection->getPassword().'";'.$command.' 2>&1 | grep -v "Using a password" || true', $result_code);
```

Reply:
@biozshock 
that line was actually leftover code, so I removed it